dbt_sugar/core/documentation/documentation_helper.py

```diff
@@ -0,0 +1,159 @@
+"""Contains documentation related utils."""
+import logging
+import os
+from pathlib import Path
+from typing import Any, Dict, List
+
+from dbt_sugar.core.yaml_module.yaml_helpers import open_yaml, save_yaml
+
+COLUMN_NOT_DOCUMENTED = "No description for this column."
+MODEL_NOT_DOCUMENTED = "No description for this model."
+
+logging.basicConfig(level=logging.INFO)
+
+
+class Documentation:
+    """Class to administrate the documentation in DBT."""
+
+    def __init__(self, repository_path: Path) -> None:
+        self.repository_path = repository_path
+        self.dbt_definitions: Dict[str, str] = {}
+        self.save_all_descriptions()
+
+    def get_column_description_from_dbt_definitions(self, column_name: str) -> str:
+        """Searches for the description of a column in all the descriptions in DBT.
```

Comment:
I'm not sure I understand what this does. Are you thinking of adding functionality to this later? Right now it doesn't seem to get any definitions nor to search but simply returns the documentation empty placeholder.

Reply:
We have a global dictionary object that we create in the constructor, to now all the definitions that are in DBT, with
{column_name: column_description}, I just added this method to not repeat the same line all over the place, of if it doesn't find the column_name return empty description.